tests/common.go

```diff
@@ -84,6 +87,27 @@ func InitInstance() {
 
 	err = Inst().N.Init()
 	expect(err).NotTo(haveOccurred())
+
+	nodes := node.GetWorkerNodes()
+	expect(nodes).NotTo(beEmpty())
+
+	switch nodes[0].PlatformType{
+	case node.PlatformIKS:
+		specFactory, err := spec.NewFactory(fmt.Sprintf("%s/ssh", defaultSpecsRoot), Inst().S)
+		expect(err).NotTo(haveOccurred())
+		dsSpec, err := specFactory.Get("ssh")
```

Comment:
instead of ssh, we can call it debug since actually it's not doing ssh but rather nsenter.